docs/teams.md

```diff
@@ -30,6 +30,7 @@ The `ApplicationTeam` will do the following:
 - Register IAM users for cross-account access
 - Create a shared role for cluster access. Alternatively, an existing role can be supplied.
 - Register provided users/role in the `awsAuth` map for `kubectl` and console access to the cluster and namespace.
+- (Optionally) read all network policy manifest files stored in a provided directory, and applies them.
```

Comment:
Similar to above comment, let's not state that those are network policy only. Read all additional manifests (e.g., network policies, OPA policies, other).

Reply:
How is it resolved?